tests/unit/test_repo.py

```diff
@@ -69,8 +73,9 @@ def test_repo_unlink(qapp, qtbot, monkeypatch):
 
 def test_password_autofill(qapp, qtbot):
     main = qapp.main_window
-    main.repoTab.new_repo()  # couldn't click menu
-    add_repo_window = main.repoTab._window
+    tab = main.repoTab
```

Comment:
For consistency across the tests, I defined `main` and `tab` in each test that used them.